rdt/transformers/utils.py

```diff
@@ -1,10 +1,10 @@
 """Tools to generate strings from regular expressions."""
 
 import re
-import sre_parse
 import string
 
 import numpy as np
+import sre_parse
```

Comment:
I think this causes builds to fail depending on the machine. I think on sdv we just ended up ignoring it

Reply:
I'll fix it in my other RDT PR.